README.md

```diff
@@ -372,6 +374,713 @@ As an alias:
 alias unlicense='license-up unlicense'
 ```
 
+### GPL
+
+#### Version 3
+
+To create a [GNU General Public License version 3](https://oss.ninja/gpl-3.0), run:
+
+```Bash
+license-up gpl3
+```
+
+Here is a working example of it.
+
+```Bash
+license-up gpl3
+```
+
+Will create a file `LICENSE` with following content:
```

Comment:
I think we shouldn't put the entire GNU license in README. It's way too long. Perhaps we can truncate to only show the top and mention it?

Reply:
There's really two options here ... we can include all of the text, but have it hidden like this:

<details><summary>Some hidden text</summary>
All of the hidden content
</details>

Or we can just include the [header](https://oss.ninja/gpl-3.0-header) of the license

Reply:
The hidden text is nice. Let's do that for all other licences too. I think it would be a good chance.

Reply:
Sounds good ... I'll do it for this one, and then create a new pull request for all of the other ones